models/appModels.js

```diff
@@ -1,7 +1,21 @@
 const db = require('../db/connection');
 
 exports.getAllCategories = () => {
-  return db.query(`SELECT * FROM categories`).then(({ rows: categories }) => {
+  return db.query(`SELECT * FROM categories;`).then(({ rows: categories }) => {
     return categories;
   });
 };
+
+exports.getAllReviews = () => {
+  return db
+    .query(
+      `SELECT owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, CAST(COUNT(comment_id) AS INT) AS comment_count FROM reviews
```

Comment:
Awesome use of the cast and count aggregate functions in order to get the count back as a number we can use on the front end 👍